### PR TITLE
Feature unsub component

### PR DESCRIPTION
**Pull request by franckbushbaum**
The page for that will list the reasons why users have unsubscribed.

### CODE REVIEW

File: src/redux/sagas/unsubscribe.saga.js [hunk 1]
```diff
@@ -6,15 +6,14 @@ function* fetchUnsubFeedback(action) {
     console.log('Get all subs saga', action.payload);
     // we are sending form data to the database
     const response = yield axios.post("/api/feedback", action.payload);
-    // yield put({ type: "FETCH_UNSUB_FEEDBACK", payload: response.data });
-    console.log('response is:', response.data)
+    // yield put({ type: "SET_UNSUB_FEEDBACK", payload: response.data });
```
Comment (gTambo) [inline src/redux/sagas/unsubscribe.saga.js:9]: This is still commented out, so why did you change it?

File: src/redux/reducers/unsubFeedback.reducer.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const unsubFeedbackReducer = (state = [], action) => {
-    // console.log('inside REDUCER, the action.payload is:', action.payload)
+    // console.log('inside UNSUBFEEDBACK REDUCER, the action.payload is:', action.payload)
```
Comment (gTambo) [inline src/redux/reducers/unsubFeedback.reducer.js:2]: Does this ever get called from anywhere?

File: src/redux/sagas/unsubscribe.saga.js (hunk 1 above)
Comment (franckbushbaum) [reply src/redux/sagas/unsubscribe.saga.js:9]: Original plan was make the GET request on line 9, use FETCH_UNSUB to send to a reducer (since I believe FETCH_UNSUB is not being used) So I changed the name to SET_UNSUB, decided it was clearer to make a whole new saga instead, but didn't change the name back.

**Comment by franckbushbaum** [comment]
No, I only see it exists here and in the rooter reducer

Screen Shot 2021-12-02 at 3 41 34 PM
.

**Review by gTambo — APPROVED** [review]
